app/models/feeds/apple_subscription.rb

```diff
@@ -20,7 +26,7 @@ class Feeds::AppleSubscription < Feed
   def set_defaults
     self.slug ||= DEFAULT_FEED_SLUG
     self.title ||= DEFAULT_TITLE
-    self.audio_format ||= DEFAULT_AUDIO_FORMAT
+    self.audio_format ||= guess_audio_format
```

Comment:
Now that this is variable, should we also open it up in the UI to change?

Reply:
Done!